ui/app/serializers/clients/activity.js

```diff
@@ -1,6 +1,58 @@
 import ApplicationSerializer from '../application';
 import { formatISO } from 'date-fns';
 
+export default ApplicationSerializer.extend({
+  flattenDataset(payload) {
+    // let topTen = payload ? payload.slice(0, 10) : [];
+    return payload?.map((ns) => {
+      // 'namespace_path' is an empty string for root
+      if (ns['namespace_id'] === 'root') ns['namespace_path'] = 'root';
```

Comment:
Not sure why we need to set this if we're defaulting to the `namespace_id` for label if `namespace_path` is falsey below.

Reply:
ahh yes - i removed it in the `monthly.js` serializer but forgot to do the same here. originally, i was worried `path` could be empty. but when reviewing our docs that appears to not be allowed :)